packages/benchmarks/tsconfig.json

```diff
@@ -0,0 +1,26 @@
+{
+  "compilerOptions": {
+    "target": "es2017",
```

Comment:
We're using ES2019 in the other packages

Reply:
Done, copied over lit-html's config